openfl-tutorials/interactive_api/GAUDI/PyTorch_Kvasir_UNet/README.md

```diff
@@ -0,0 +1,93 @@
+# PyTorch_Kvasir_UNet
+
+## **Habana Tutorials**
+#### The name of the file/example that contain HPU adaptations start with "HPU". 
+For example: PyTorch_Kvasir_UNet.ipynb placed under workspace folder contains the required HPU adaptations.
+
+ All the execution steps mention in last section (**III. How to run this tutorial**) remain same for HPU examples but as pre-requisite it needs some additional environment setup and Habana supported package installations which is explained below in **section I and II**.
+
+ **Note:** By default these experiments utilize 1 HPU device
+ <br/>
+
+ ## **I. Habana Setup**
+
+ Please follow the instructions provided in the [Gaudi Installation Guide](https://docs.habana.ai/en/latest/Installation_Guide/index.html) to set up the environment including the ```$PYTHON``` environment variable.
+ <br/>
+ 
+ This example was tested on AWS DL1 EC2 instance created by following the instructions mentioned [here](https://docs.habana.ai/en/latest/AWS_EC2_DL1_and_PyTorch_Quick_Start/AWS_EC2_DL1_and_PyTorch_Quick_Start.html) . 
```

Comment:
Add a note: :"The example can be executed on any Gaudi configuration, DL1 on AWS or on an on-prem configuration. Currently, the validation has only been done on DL1 instance on AWS".